django/test/client.py

```diff
@@ -421,93 +425,126 @@ def _get_path(self, parsed):
         # Refs comment in `get_bytes_from_wsgi()`.
         return path.decode("iso-8859-1")
 
-    def get(self, path, data=None, secure=False, **extra):
+    @staticmethod
+    def _transform_headers(headers):
+        return {
+            HttpHeaders.to_header_name(header_name): value
```

Comment:
Not 100% sure of exposing `HttpHeaders` so that we can use one method in `RequestFactory`.
But at the same time, it's nicer if we can re-use its internal attributes `HttpHeaders.HTTP_PREFIX` and `HttpHeaders.UNPREFIXED_HEADERS`.